src/landing-pages/shopify/index.js

```diff
@@ -0,0 +1,105 @@
+import React from "react";
+import "./index.scss";
+import imglaptop from "./images/imgfooterHeader.png";
+import logo from "./images/logo.jpg";
+import { ReactComponent as Logoshop } from "./images/logoshopify.svg";
+import { ReactComponent as Icon2 } from "./images/svgexport-2.svg";
+import { ReactComponent as Icon3 } from "./images/svgexport-3.svg";
+import { ReactComponent as Icon4 } from "./images/svgexport-4.svg";
+
+export function renderHeader() {
+  return (
+    <div className="Header">
+      <a href="/shopify">
+        <Logoshop className="Logo" />
+      </a>
+      <h1>Sell online with Shopify</h1>
+      <h2>Trusted by over 600,000 businesses worldwide</h2>
+      <div className="Email-textbox-button-text">
```

Comment:
learn how to name a className. It should be in the same format (UPPERCASE, lowercase, CapitalizationForName, underscore_for_name, hyphen-for-name). Choose one of them, and use it for the whole website